diff --git a/node_modules/dressed/.bun-tag-5d490c66d51a3393 b/.bun-tag-5d490c66d51a3393
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/node_modules/dressed/.bun-tag-d834501b95ccf064 b/.bun-tag-d834501b95ccf064
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/dist/server/build/build.js b/dist/server/build/build.js
index e42666f3385b5e6932ffb7ccb1d654f3d50d71ac..5d39d94f070a600febc37c6a92d04a345848bed2 100644
--- a/dist/server/build/build.js
+++ b/dist/server/build/build.js
@@ -41,7 +41,7 @@ export default async function build(config = {}, { bundle = bundleFiles } = {})
     logDefer("Bundling handlers");
     await bundle(entriesPath, ".dressed/tmp");
     const { commands, components, events } = await import(__rewriteRelativeImportExtension(resolve(entriesPath.replace(".ts", ".mjs"))));
-    console.log(); // This just adds a newline before the logged trees for consistency
+    //console.log(); // This just adds a newline before the logged trees for consistency
     return {
         commands: parseCommands(commands, `${root}/commands`),
         components: parseComponents(components, `${root}/components`),
diff --git a/dist/server/build/log-tree.js b/dist/server/build/log-tree.js
index 290188faad12727cbe45cd6211f1c209be204c4c..0a90ff00b5e699c968d0a6d419fae19fb66f1ffa 100644
--- a/dist/server/build/log-tree.js
+++ b/dist/server/build/log-tree.js
@@ -38,7 +38,7 @@ export default function logTree(...titles) {
                     .join("  ");
                 lines.push(`${i === 0 ? " " : i === 1 ? (rowCount === 2 ? "-" : "┌") : i === rowCount - 1 ? "└" : "├"} ${chopped.has(i) ? "\x1b[9m" : ""}${row}\x1b[0m`, ...((_a = asides[i]) !== null && _a !== void 0 ? _a : []));
             }
-            lines.concat("").map((l) => console.log(l));
+            //lines.concat("").map((l) => console.log(l));
         },
     };
 }
diff --git a/dist/utils/log.js b/dist/utils/log.js
index 378173742208e7494d5f9292a16ccb5cc2157a61..f99f7ccf51c7a492c96012f7c2435903d603ceac 100644
--- a/dist/utils/log.js
+++ b/dist/utils/log.js
@@ -1,7 +1,7 @@
 export const errorSymbol = "\u001B[31m✖\u001B[39m";
 export const warnSymbol = "\u001B[33m⚠\u001B[39m";
-export const logDefer = (...s) => console.info("\u001B[34m…\u001B[39m", ...s);
-export const logError = (...s) => console.error(errorSymbol, ...s);
-export const logWarn = (...s) => console.warn(warnSymbol, ...s);
-export const logSuccess = (...s) => console.log("\u001B[32m✔\u001B[39m", ...s);
+export const logDefer = (...s) => {};
+export const logError = (...s) => {};
+export const logWarn = (...s) => {};
+export const logSuccess = (...s) => {};
 //# sourceMappingURL=log.js.map
\ No newline at end of file
